Model/AbstractOpsRegistry.php

```diff
@@ -79,7 +84,11 @@ public function stop($opName, $payload = [])
         }
 
         $opId = $this->getOpId();
-        $workTime = microtime(true) - $this->tsStart[$opId];
+        if (key_exists($opId, $this->tsStart)) {
```

Comment:
Hi @lewisvoncken , so sorry for my delay. I've been very busy. I planned to do this in the next days because we need to coordinate the toolbar update too.